app/src/main/java/com/internship/move/util/constants.kt

```diff
@@ -0,0 +1,3 @@
+package com.internship.move.util
+
+const val ONBOARDING_NUMBER_OF_PAGES = 4
```

Comment:
You could define this constant directly in `PagesAdapter` in a companion object.